src/core/services/theming/theming.js

```diff
@@ -153,6 +161,9 @@ function ThemingProvider($mdColorPalette) {
     extendPalette: extendPalette,
     theme: registerTheme,
 
+    registerStyles: function(styles) {
```

Comment:
I initially thought, that there will be a function, which compiles the provided style on demand as well? Thoughts?